### ISSUE TITLE
Fast Evaluation on Validation set

### ISSUE THREAD

**Issue by AdamIshay** [comphyreasoning/compositional_physics_learner#2]
Hello,

I notice that there are no “regional proposals with attribute and physical property prediction” or “dynamic predictions” included for the validation (and training) videos. Can this be made available to do fast evaluation on the validation set? Though these are included for the test videos, the ground-truth label is not included in `test.json`, so I cannot evaluate the accuracy there either. (FYI, for the test set I notice that 25 multiple choice test questions have no choices, which end up counting as a correct question because of this). 

Thank you!

**Comment by zfchenUnique** [comment]
Dear Adam,

        Thanks for your interest on the benchmark. Sorry for the late response. The evaluation server should be ready with a few days on eval AI. I may also find the result files of the training set and validation set if possible.

Regards,
Zhenfang

**Comment by zfchenUnique** [comment]
Hi Adam,

      Just a few updates. 
       I have updated the evaluation scripts, which you you can run https://github.com/zfchenUnique/compositional_physics_learner/blob/main/merge_raw_results.py to prepare a json file that has a similar format like https://github.com/zfchenUnique/compositional_physics_learner/blob/main/test_submit.json .
        You should be able to submit the json file into the evaluation server on eval.ai. It should have be set up. However, due to the low resources allocated to the challenge, currently the server has some problems. I have raised the issues to the maintainer of eval.ai several times, it should work soon.
        If you would like to know the reuslts of your program as soon as possible, you may prepare  a json file and sent it to me.  I could evaluate it for you and show you the results.
        I will update the link when it is ready.
        I did not keep the original copies of the input files for the training set and validation set. Thus, it is a pity that I could not provide those input for you.

Regards,
Zhenfang
